GOVERNANCE.md

```diff
@@ -64,7 +64,7 @@ If you have points that you want to discuss, please send your points to the [mai
 
 ## Agenda
 
-1. General: Introduction of new participations, 42 mins
+1. General: Introduction of new participators, 42 mins
```

Comment:
participants is more established as a word I guess, but participator exists as well and is synonymous (to my knowledge).